### ISSUE TITLE
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections

### ISSUE THREAD

**Issue by homocomputeris** [spyder-ide/spyder#6280]
## Description

**What steps will reproduce the problem?**
No idea. The error report appeared unexpectedly.

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 906, in <lambda>
    clear_variables))
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 948, in execute_code
    sw.sig_prompt_ready.disconnect()
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 906, in <lambda>
    clear_variables))
  File "/usr/lib/python3.6/site-packages/spyder/plugins/ipythonconsole.py", line 948, in execute_code
    sw.sig_prompt_ready.disconnect()
TypeError: disconnect() failed between 'sig_prompt_ready' and all its connections

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.4
* Qt Versions: 5.10.0, PyQt5 5.9.2 on Linux

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.21.0 (OK)
numpy >=1.7      :  1.14.0 (OK)
sphinx >=0.6.6   :  1.6.6 (OK)
rope >=0.9.4     :  0.10.6 (OK)
jedi >=0.9.0     :  0.11.1 (OK)
psutil >=0.3     :  5.4.1 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.27.3 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.2.1 (OK)
pylint >=0.25    :  1.8.1 (OK)

```

**Comment by ccordoba12** [comment]
This was fixed in our 3.2.5 version. Please update.

**Comment by CAM-Gerlach** [comment]
You can do so with ``conda update spyder`` from the command line, by the way.
